st3/sublime_lib/syntax.py

```diff
@@ -1,38 +1,52 @@
 import sublime
 
 import re
+from collections import namedtuple
 
 __all__ = ['list_syntaxes', 'get_syntax_for_scope']
 
-class SyntaxInfo():
-    __slots__ = [ 'path', 'name', 'scope', 'hidden' ]
-    
-    def __init__(self, path, name=None, scope=None, hidden=False):
-        self.path = path
-        self.name = name
-        self.scope = scope
-        self.hidden = hidden
+
+SyntaxInfo = namedtuple('SyntaxInfo', ['path', 'name', 'scope', 'hidden'])
+SyntaxInfo.__new__.__defaults__ = (None, None, False)  # type: ignore
+
+
+def _parse_yaml_value(value):
+    if value.startswith("'"):
+        return value[1:-1].replace("''", "'")
+    elif value.startswith('"'):
+        # This is a simplifcation but one that should do for now
```

Comment:
Can we just use `sublime.decode_value` here? I'd have to compare the JSON and YAML specs to be sure, but I think this should work.

Reply:
Good idea. Yes, that should work better as it supports unicode escapes.